sensor_msgs/Image image_sub
nav_msgs/Odometry GPS_sub
---
float64[] x
float64[] y
